app/assets/javascripts/area_served.js

```diff
@@ -0,0 +1,45 @@
+
+
+
+function calculate_client_share_total(){
+    total = 0;
+    console.log("*********mark 1 total is: " +total);
+    const client_shares = document.querySelectorAll('*[id$="client_share"]')
+    client_shares.forEach(
+        cs => {
+            if(!cs.value || cs.offsetWidth === 0){
+                return;
+            }
+            if(cs.value){
+                total += parseInt(cs.value)
+            }
+        }
+    );
+    console.log("*********total is: " +total);
+    document.getElementById("partners-served-areas-client-share-total").innerHTML = total + " %";
+
+
+
+    if(total == 0 || total == 100){
+        document.getElementById("partners-served-areas-client-share-total-warning").style.visibility= 'hidden';
+        //      document.getElementById("profile-update-button").style.visibility = 'visible';
+        //    document.getElementById("profile-update-button-disabled").style.visibility = 'hidden';
+        //  document.getElementById("partner-county-client-share-total-warning-footer").style.visibility= 'hidden';
```

Comment:
Please remove commented out code if it's not going to be used.

Reply:
Also removed in next push